### ISSUE TITLE
feature request: label options and save/load

### ISSUE THREAD

**Issue by alexweisberg** [gamcil/clinker#6]
Clinker is great and I've already found a great use for it in my research. Its very easy to use and makes really nice results.

I have two requests for features. One is that it would be very nice to be able to select the label that is displayed. Currently it seems to default to a hierarchy of protein_id and then locus_tag. It would be nice to be able to select which is displayed, especially if I could choose the "gene=" tag if present. Alternatively, a table linking locus_tags to a custom label as input would be nice.

The second request is to be able to save and load the visualization state. It would be nice to be able to run the analysis on a headless cluster and then load it in the browser to visualize. Likewise it would be nice to be able to save the visualization state to a file and reload it later to make further changes. Edit: I just realized this is possible by using the -p option to save to an html file. I should have tried that first!

**Comment by gamcil** [comment]
Thanks for using it :) Glad the second problem worked itself out.

First one is on the to do list, especially given some of the trouble with identifiers others are having (https://github.com/gamcil/clinker/issues/4). I guess the best way would be to just read in every possible identifier and have a drop down to select which one to use in the visualisation?

**Comment by alexweisberg** [comment]
That would be perfect. That way one can flip through them to identify the right gene or ID even if they are annotated differently between strains.

**Comment by gamcil** [comment]
With 0.0.7, clinker now saves all name qualifiers and has a right click menu for genes where labels can be swapped and edited.

**Comment by alexweisberg** [comment]
Great! Thank you for adding this.

**Comment by alexweisberg** [comment]
I just tested this, and right clicking works to change individual genes. However, it would be nice to set the default for all genes at once, ie a dropdown in the main menu on the right to choose which item is the source of the label, like "gene" or "locus_tag". That way you dont have to individually rename every gene. I think that is what the "Label type" dropdown is supposed to be? However the dropdown dialog is empty when I tried running it. 
![clinker](https://user-images.githubusercontent.com/12143265/100476677-0f519100-309b-11eb-90cf-871f9a86d4f9.png)
